tools/queuebench/go.mod

```diff
@@ -0,0 +1,48 @@
+module github.com/elastic/apm-queue/cmd/queuebench
```

Comment:
We used to have multiple modules -- we had one in systemtest and removed it: https://github.com/elastic/apm-queue/pull/214. We removed that one because it was causing issues with Renovate.

So with that said, can we avoid introducing this module? IIANM the module dependencies you have added that aren't in the main apm-queue module are `pflag` and `uitable`. I think we could get by without those for now to keep things simple as a whole. WDYT?

Reply:
sure